draft-ietf-gnap-resource-servers.md

```diff
@@ -0,0 +1,502 @@
+---
+title: 'Grant Negotiation and Authorization Protocol Resource Server Connections'
+docname: draft-ietf-gnap-resource-servers-latest
+category: std
+
+ipr: trust200902
+area: Security
+workgroup: GNAP
+keyword: Internet-Draft
+
+stand_alone: yes
+pi: [toc, tocindent, sortrefs, symrefs, strict, compact, comments, inline, docmapping]
+
+author:
+  - ins: J. Richer
+    name: Justin Richer
+    organization: Bespoke Engineering
+    email: ietf@justin.richer.org
+    uri: https://bspk.io/
+    role: editor
+
+  - ins: A. Parecki
+    name: Aaron Parecki
+    email: aaron@parecki.com
+    organization: Okta
+    uri: https://aaronparecki.com
+
+  - ins: F. Imbault
+    name: Fabien Imbault
+    organization: acert.io
+    email: fabien.imbault@acert.io
+    uri: https://acert.io/
+
+normative:
```

Comment:
Probably should review that list, I guess there's a lot we don't use in here specifically.

Reply:
Yes, and the tools will throw warnings for unused references that we can use to trim things -- I just left the list as-is for now as we settle on which text goes where to avoid compilation errors for now.